src/logic/operate.js

```diff
@@ -0,0 +1,26 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '−') {
+    return one.minus(two).toString();
+  }
+  if (operation === '×') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
+  }
+  throw Error(`Unknown operation '${operation}'`);
+}
```

Comment:
- [ ] Your `operate.js` file is not the same as the provided file. As I can see there is no exception handler for the modulus operation for `0`. Kindly ensure your file is the same as this: [`operate.js file`](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/operate.js)